### PR TITLE
Add form section

### PR DESCRIPTION
**Pull request by hiwina**
Footer section is added to add contact form.
Added form tag in the footer tag.
Added list for social tabs.
Styled the index with css.
Worked with my coding partner.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -183,7 +184,45 @@ <h4>Skills</h4>
...
+                    <input type="text"  id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+                    <input type="email"  id="email" name="user_email"  placeholder="Email address" required>
+                    <textarea type="text"  id="message" name="user_message"  maxlength="500" required></textarea>
+                </div>
+                <div class="back-img4"></div>
+                <div class="get-in-touch-btn">
+                    <button class="form-btn" type="submit">Get in touch </button>
+                </div>
+            
+            </form>
```
Comment (Santiago220991) [inline index.html]: - Kindly replace the `footer` tag with a `section` tag. The `footer` tag defines a footer for a document or section. A footer element typically contains authorship information, copyright information, contact information, sitemap, back to top links. The footer should not contain a form. :nerd_face: :nerd_face: 

- Kindly make sure that your project sticks as much as possible to the Figma template provided. Please add the placeholder to the `textarea` tag as the Figma template shows. :pray: :pray: 

Your design:
Screenshot from 2022-10-05 07-10-33

Figma design:
Screenshot from 2022-10-05 07-12-42

File: index.html [hunk 2]
```diff
@@ -183,7 +184,45 @@ <h4>Skills</h4>
...
+                 </ul> 
+                 <div class="footer-email">
+                    <a href="mailto:hiwina@gmail.com" id="">hiwina@gmail.com</a>                
+               </div>
+            </div>
+            
+            <div>
+                <img src="image/Light.png" alt="footer image" id="footer-image">
+
+            </div>
```
Comment (Santiago220991) [inline index.html]: - According to the comment above. Please wrap this information inside a footer tag. :+1: :smiley: :smiley: 

- Kindly make sure that all your social media icons are rendering correctly. There is an issue with the LinkedIn icon please fix it. :eyes: :eyes: 

image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @hiwina,
I'm Santiago your reviewer for this PR. :wave: :wave:  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
:heavy_check_mark: Good PR summary.
:heavy_check_mark: Followed GitHub flow.
:heavy_check_mark: No linter errors.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅
## Hi @hiwina 👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**

 

## Highlights:
* Well written PR description ✔️ 
* Proper use of GitHub flow ✔️  
* All tests passed ✔️ 

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**